sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_client.py

```diff
@@ -40,13 +41,13 @@ class ConversationAnalysisClient(
      https://:code:`<resource-name>`.cognitiveservices.azure.com). Required.
     :type endpoint: str
     :param credential: Credential needed for the client to connect to Azure. Required.
-    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :type credential: ~azure.core.credentials.TokenCredential
     :keyword api_version: Api Version. Default value is "2022-05-01". Note that overriding this
      default value may result in unsupported behavior.
     :paramtype api_version: str
     """
 
-    def __init__(self, endpoint: str, credential: AzureKeyCredential, **kwargs: Any) -> None:
+    def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any) -> None:
```

Comment:
Why is the type quoted?

Reply:
https://github.com/Azure/autorest.python/issues/1326